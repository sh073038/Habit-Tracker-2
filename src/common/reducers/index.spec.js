import reducer from './index'

describe('Root reducer', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })
})
