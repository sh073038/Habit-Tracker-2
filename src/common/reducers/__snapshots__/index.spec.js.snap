// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root reducer should return the initial state 1`] = `
Object {
  "dashboard": Object {
    "selectedUser": null,
  },
  "firebase": Object {
    "auth": Object {
      "isEmpty": true,
      "isLoaded": false,
    },
    "authError": null,
    "data": Object {},
    "errors": Array [],
    "isInitializing": false,
    "listeners": Object {
      "allIds": Array [],
      "byId": Object {},
    },
    "ordered": Object {},
    "profile": Object {
      "isEmpty": true,
      "isLoaded": false,
    },
    "requested": Object {},
    "requesting": Object {},
    "timestamps": Object {},
  },
}
`;
